From eb267a661a1636e69969d210654d90513fe81c2f Mon Sep 17 00:00:00 2001
From: Jeff Moyer <jmoyer@redhat.com>
Date: Fri, 3 May 2024 17:05:49 -0400
Subject: [PATCH 3/3] man: fix IORING_REGISTER_RING_FDS docs

The documentation for the 'arg' parameter is incorrect.  Fix it.

Origin: upstream, commit:eb267a661a1636e69969d210654d90513fe81c2f
Forwarded: not-needed
Signed-off-by: Jeff Moyer <jmoyer@redhat.com>
Link: https://lore.kernel.org/r/x49bk5mehci.fsf@segfault.usersys.redhat.com
Signed-off-by: Jens Axboe <axboe@kernel.dk>
---
 man/io_uring_register.2 | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/man/io_uring_register.2 b/man/io_uring_register.2
index fbfae2a..4590588 100644
--- a/man/io_uring_register.2
+++ b/man/io_uring_register.2
@@ -528,8 +528,8 @@ of the ring file descriptor itself. This reduces the overhead of the
 system call.
 
 .I arg
-must be set to an unsigned int pointer to an array of type
-.I struct io_uring_rsrc_register
+must be set to a pointer to an array of type
+.I struct io_uring_rsrc_update
 of
 .I nr_args
 number of entries. The
@@ -570,8 +570,8 @@ Unregister descriptors previously registered with
 .B IORING_REGISTER_RING_FDS.
 
 .I arg
-must be set to an unsigned int pointer to an array of type
-.I struct io_uring_rsrc_register
+must be set to a pointer to an array of type
+.I struct io_uring_rsrc_update
 of
 .I nr_args
 number of entries. Only the
-- 
2.43.0

